Enable aarch64 for fixing undefined symbol error.

jasonjk-park opened the pull request:
Summary: ARM can be safely supported

Reviewed By: andrewjcg, aaronenyeshi

Differential Revision: D49921679

aaronenyeshi reviewed (approved):
Thanks for the quick fix!

jasonjk-park committed:
Enable aarch64 for fixing undefined symbol error. (#110542)

Summary:

ARM can be safely supported

Reviewed By: andrewjcg, aaronenyeshi

Differential Revision: D49921679
